tests/e2e/unified_test.go

```diff
@@ -155,66 +155,34 @@ func TestAllInSeries(t *testing.T) {
 					}
 
 					expectedPath := filepath.Join(fixture.SourceDir, "export.yaml")
-					output := h.MustReadFile(outputPath)
-					h.CompareGoldenFile(expectedPath, string(output), h.IgnoreComments, h.ReplaceString(project.ProjectID, "example-project-id"))
+					output, err := os.ReadFile(outputPath)
```

Comment:
I think this is what MustReadFile did ... any reason to remove it?

Reply:
No big difference, but I do not think it's necessary to turn that into a new function? Handle the error after `ReadFile` is called seems a normal pattern and more straightforward.